### PR TITLE
add \BREAK & \CONTINUE

### PR DESCRIPTION
**Pull request by leovan**
add \BREAK for loop.

### CODE REVIEW

**Comment by SaswatPadhi** [comment]
Great! Thank you.

The changes look good to me and I would be happy to merge them. While at it though, would you also like to add `\CONTINUE`?

**Comment by leovan** [comment]
Sure, I used `\BREAK` in my post once, but forget the similar keyword `\CONTINUE`. I will make this done later on.

**Comment by SaswatPadhi** [comment]
Perfect. Thank you so much. I'll wait for you to push that change and then merge it together.

**Comment by leovan** [comment]
I think all done, please have a check @SaswatPadhi .

File: src/Parser.js [hunk 1]
```diff
@@ -354,7 +356,7 @@ Parser.prototype._parseRepeat = function() {
 };
 
 var INPUTS_OUTPUTS_COMMANDS = ['ensure', 'require', 'input', 'output'];
-var STATEMENT_COMMANDS = ['state', 'print', 'return'];
+var STATEMENT_COMMANDS = ['state', 'print', 'return', 'break', 'continue'];
 Parser.prototype._parseCommand = function(acceptCommands) {
```
Comment (SaswatPadhi) [inline src/Parser.js:378]: This function is called for all `STATEMENT_COMMANDS` at line 241.

However, the behavior of `break` and `continue` commands is slightly different from `print`, `state` and `return`. `break` and `continue` do not accept any arguments -- they should not have `<open-text>` after them.

In this function, line 365 inserts an `<open-text>` child node for all the statement commands, but that shouldn't happen for `break` and `continue`.

**Review by SaswatPadhi — CHANGES_REQUESTED** [review]

File: src/Parser.js (hunk 1 above)
Comment (leovan) [reply src/Parser.js:378]: Yes, thought it works without `<open-text>` after `\BREAK` and `\CONTINUE`, we also should treat them as new type of commands. I update the code, please check whether it's a better way to do so.

**Comment by SaswatPadhi** [comment]
@leovan Sorry about the delay in merging this PR. The changes were good. I only made a few minor improvements. Thanks again!

**Pull request merged**